docs/pages/docs/apis/filters.md

```diff
@@ -74,29 +74,29 @@ The `json` field type does not support filters.
 - If the `type` is `string`(the default), the same filters as `text` will be available.
 - If the `type` is `integer`, the same filters as `integer` will be available.
 - If the `type` is `enum`, the following filters will be available:
-  | **Filter name** | **Type** | **Description** |
-  | --------------- | ---------- | ------------------- |
-  | `equals` | `ListKeyFieldKeyType` | Equals |
-  | `in` | `[ListKeyFieldKeyType!]` | Is in the array |
-  | `notIn` | `[ListKeyFieldKeyType!]` | Is not in the array |
-  | `not` | `ListKeyFieldKeyTypeNullableFilter` | Does not match the inner filter |
+  \| **Filter name** \| **Type** \| **Description** \|
+  \| --------------- \| ---------- \| ------------------- \|
+  \| `equals` \| `ListKeyFieldKeyType` | Equals |
+  \| `in` \| `[ListKeyFieldKeyType!]` | Is in the array |
+  \| `notIn` \| `[ListKeyFieldKeyType!]` | Is not in the array |
+  \| `not` \| `ListKeyFieldKeyTypeNullableFilter` | Does not match the inner filter |
```

Comment:
As seen in https://github.com/keystonejs/keystone/pull/7895, this is completely broken